### PR TITLE
UI - cleanup unused code (azure worker node settings, disabled unit tests)

### PR DESCRIPTION
**Pull request by miclettej**
This PR is general house keeping to remove some old unused code in the UI.

Renames AWS_GET_NO_PROXY_INFO to NETWORK_STEP_GET_NO_PROXY_INFO since it is used across multiple providers 

### Describe testing done for PR

UI linting, build and manual testing on local dev environment. Confirmed that No Proxy info is getting populated as expected.

### PR Checklist

- [x] Squash the commits into one or a small number of logical commits
      
- [x] Use good commit [messages](https://github.com/vmware-tanzu/tanzu-framework/blob/main/CONTRIBUTING.md)
- [x] Ensure PR contains terms all contributors can understand and links all contributors can access

### CODE REVIEW

**Review by swalner-vmware — APPROVED** [review]
Looks good to me

File: pkg/v1/tkg/web/src/app/views/landing/azure-wizard/provider-step/azure-provider-step.component.spec.ts [hunk 1]
```diff
@@ -146,85 +146,4 @@ describe('AzureProviderStepComponent', () => {
...
-    //     const connectBtn = fixture.debugElement.query(By.css("button.btn-primary"));
-    //     connectBtn.nativeElement.click();
-    //     fixture.whenStable().then(
-    //         () => {
-    //             fixture.detectChanges();
-    //             expect(component.errorNotification).not.toBeFalsy();
-    //             expect(component.validCredentials).toBeFalsy();
-    //         }
-    //     );
-    // }));
```
Comment (anujc25) [inline pkg/v1/tkg/web/src/app/views/landing/azure-wizard/provider-step/azure-provider-step.component.spec.ts:228]: If this is not needed, should we just remove this? Just wanted to confirm this is commented intentionally and not kept by mistake.

**Review by anujc25 — APPROVED** [review]
LGTM. Just one comment to confirm the commented section

File: pkg/v1/tkg/web/src/app/views/landing/azure-wizard/provider-step/azure-provider-step.component.spec.ts (hunk 1 above)
Comment (miclettej) [reply pkg/v1/tkg/web/src/app/views/landing/azure-wizard/provider-step/azure-provider-step.component.spec.ts:228]: Yes exactly, it's not super clear looking at the code review but my change is actually removing all of those unused tests

**Pull request merged**